mock-abis-default.properties

```diff
@@ -8,8 +8,14 @@ management.endpoint.health.show-details=always
 management.endpoits.web.exposure.include=info,health,refresh
 server.port=8081
 abis.return.duplicate=false
-secret_url=${mosip.kernel.authmanager.url}/v1/authmanager/authenticate/clientidsecretkey
-secret_url.clientnId=mosip-regproc-client
-secret_url.id=string
-secret_url.secretKey=${mosip.regproc.client.secret}
-secret_url.appId=regproc
+#iam adapter
+auth.server.admin.issuer.uri=${keycloak.external.url}/auth/realms/
+mosip.iam.adapter.appid=regproc
+mosip.iam.adapter.clientid=mosip-regproc-client
+mosip.iam.adapter.clientsecret=${mosip.regproc.client.secret}
+mosip.kernel.auth.adapter.ssl-bypass=true
+mosip.kernel.auth.appid-realm-map={regproc:'mosip'}
+mosip.kernel.auth.appids.realm.map={regproc:'mosip'}
+#iam adapter disable local end points
+mosip.service.end-points=/**/*
+mosip.service.exclude.auth.allowed.method=GET,POST,DELETE
```

Comment:
Do we need the above 3 lines ? What do the last 2 config's mean ?

Reply:
For apis in mock abis doesnt need authentication.While calling other apis it will use token.So to exclude the local end points of abis we added .Yesterday we faced in dev2.then it has fixed by adding.